content/docs/tasks/security/vault-ca/index.md

```diff
@@ -69,53 +70,106 @@ env:
   value: "istio_ca/sign/istio-pki-role"
 {{< /text >}}
 
-## Deploy a testing workload
+## Deploy testing workloads
 
-This section deploys a testing workload `httpbin`. When the sidecar of the
+This section deploys testing workloads `httpbin` and `sleep`. When the sidecar of a
 testing workload requests a certificate through SDS, Node Agent will send
 certificate signing requests to Vault.
 
-1.  Generate the deployment for an example `httpbin` backend:
+1.  Generate the deployment for an example `httpbin` backend and an example `sleep` backend:
 
     {{< text bash >}}
     $ istioctl kube-inject -f @samples/httpbin/httpbin.yaml@ > httpbin-injected.yaml
+    $ istioctl kube-inject -f @samples/sleep/sleep.yaml@ > sleep-injected.yaml
     {{< /text >}}
 
-1.  Deploy the example backend:
+1.  Create a service account `vault-citadel-sa`:
 
     {{< text bash >}}
-    $ kubectl apply -f httpbin-injected.yaml
+    $ kubectl create serviceaccount vault-citadel-sa
     {{< /text >}}
 
-1.  List Node Agent's pods:
+1.  Edit the service account `vault-citadel-sa` to use an example JWT token that has been configured
+on the testing Vault CA for authentication and authorization:
 
     {{< text bash >}}
-    $ kubectl get pod -n istio-system -l app=nodeagent -o jsonpath={.items..metadata.name}
+    $ export SA_SECRET_NAME=$(kubectl get serviceaccount vault-citadel-sa -o=jsonpath='{.secrets[0].name}')
+    $ kubectl edit secret ${SA_SECRET_NAME}
+
+    # When editing the secret, change the field "token" to: ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklpSjkuZXlKcGMzTWlPaUpyZFdKbGNtNWxkR1Z6TDNObGNuWnBZMlZoWTJOdmRXNTBJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5dVlXMWxjM0JoWTJVaU9pSmtaV1poZFd4MElpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WldOeVpYUXVibUZ0WlNJNkluWmhkV3gwTFdOcGRHRmtaV3d0YzJFdGRHOXJaVzR0Y21aeFpHb2lMQ0pyZFdKbGNtNWxkR1Z6TG1sdkwzTmxjblpwWTJWaFkyTnZkVzUwTDNObGNuWnBZMlV0WVdOamIzVnVkQzV1WVcxbElqb2lkbUYxYkhRdFkybDBZV1JsYkMxellTSXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMblZwWkNJNklqSXpPVGs1WXpZMUxUQTRaak10TVRGbE9TMWhZekF6TFRReU1ERXdZVGhoTURBM09TSXNJbk4xWWlJNkluTjVjM1JsYlRwelpYSjJhV05sWVdOamIzVnVkRHBrWldaaGRXeDBPblpoZFd4MExXTnBkR0ZrWld3dGMyRWlmUS5STkgxUWJhcEpLUG1rdFYzdENucGl6N2hvWXB2MVRNNkxYelRoT3RhRHA3TEZwZUFOWmNKMXpWUWR5czNFZG5sa3J5a0dNZXBFanNkTnVUNm5kSGZoOGpSSkFadU5XTlBHcmh4ejRCZVVhT3FaZzN2N0F6SmxNZUZLallfZmlUWVlkMmdCWlp4a3B2MUZ2QVBpaEhZbmcyTmVOMm5LYmlaYnNuWk5VMXFGZHZiZ0NJU2FGcVRmMGRoNzVPemdDWF8xRmg2SE9BN0FOZjdwNTIyUERXX0JSbG4wUlR3VUpvdkNwR2VpTkNHZHVqR2lOTERaeUJjZHRpa1k1cnlfS1hUZHJWQWNUVXZJNmx4d1JiT05OZnVOOGhySURsOTV2SmpoVWxFLU8tX2N4OHFXdFhOZHFKbE1qZTFTc2lQQ0w0dXE3ME9lcEdfSTRhU3pDMm84YUR0bFE=
+    {{< /text >}}
+
+1.  Edit the `httpbin` deployment to use the service account `vault-citadel-sa`.
```

Comment:
We are only adding the special service account to the yaml. 

For better user experience, is it possible to create another http-vault.yaml / sleep-vault.yaml with the updated service account so customers don't do anything? Or can we update the secret associated with service account "default" and "sleep" the same way as "vault-citadel-sa", so we don't need to update the yaml files?

Reply:
Done: created http-vault.yaml and sleep-vault.yaml in the PR https://github.com/istio/istio/pull/12917 so customers do not need to set the service account name. 
As to update the secret associated with service account to be the same as the example JWT, I could not find a way to update the secret through a yaml configuration file.

Reply:
Oh my point was not to update secret through yaml. 

We are using 'default' service account for httpbin.yaml and 'sleep' for sleep.yaml. If we can update the secrets of service accounts 'default' and 'sleep' separately, then we don't need the extra vault-citadel-sa and the extra http-vault.yaml and sleep-vault.yaml. 

I am fine either way, but if we go with adding the new yaml files, we need to add another PR to the master branch.

Reply:
https://github.com/istio/istio/pull/12922 is created for the master branch.